packages/tsconfig-reference/copy/id/sections/compilerOptions.md

```diff
@@ -0,0 +1,3 @@
+### Pilihan - Pilihan Compiler
+
+Banyak pilihan yang membuat jumlah besar di pengaturan Typescript's dan itu juga meliputi bagaimana bahasa seharusnya bekerja.
```

Comment:
Menurut saya terjemahan untuk `these options make up the bulk of TypeScript's configuration` itu kurang lebih `Pilihan-pilihan ini berisi sebagian besar konfigurasi TypeScript`.

Menurut Mas @nusendra @mazipan @mandaputtra @isatrio gimana?